docs/source/conf.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
```

Comment:
Removing this _may_ be safe, but it was auto-generated with this coding tag, so it's probably wisest to leave it there (it signals to Python the encoding of the file).